### ISSUE TITLE
Suppress some of the printf() in the .c  codes

### ISSUE THREAD

**Issue by sguillot** [xpsi-group/xpsi#222]
Somehow in the python3 compilation of the .pyx files a lot of printf() statements become visible when the `integrator.c` or the `GlobalRayMap.c` are called. This is visible in the tutorial notebooks and will potentially show up at a lot of output during inference runs (to be checked: @yveskini ).

These outputs could be suppressed by adding a `debug=True` options in the calls for `integrator` and `GlobalRayMap` that are in `Photosphere.py`.

**Comment by thjsal** [comment]
Can you specify any of those printings? Did those exist also in Python2 version of X-PSI?

**Comment by sguillot** [comment]
In Python 2, they were showing up in the terminal and not in the notebook.

These are the printf() in the `integrator.pyx` (line 419-420), which then appear in `integrator.c`:
```
printf("\n ")
printf("\n------------------------------------------------------------->>>")
printf("\nPhase index: %d", <int>k)
printf("\nFraction of hot rays: %.6f", <double>(<double>NUM_HITS/<double>(NUM_MISSES + NUM_HITS)))
```

This is the printf() in the `globalRayMap.pyx` (line 419-420), which then appear in `globalRayMap.c`:
```
printf("\nThread %d is tracing annulus #%d of rays.", <int>T, <int>i)
```

**Comment by thjsal** [comment]
I see! At least these files and functions are not used for the inference runs. As far as I know, these are just for imaging the star using an alternative from-observer-to-star method.

**Comment by sguillot** [comment]
Resolved in PR #224
